packages/react-native-windows-init/README.md

```diff
@@ -21,6 +21,8 @@ Options:
 | `--verbose`   | Enables logging.                               | [boolean]                                        |
 | `--language`  | Which language the app is written in.          | [string] [choices: "cs", "cpp"] [default: "cpp"] |
 | `--overwrite` | Overwrite any existing files without prompting | [boolean]                                        |
+| `--useWinUI3`         | Targets WinUI 3 (through the [Windows App SDK](https://docs.microsoft.com/windows/apps/windows-app-sdk/)) instead of UWP XAML. | [boolean] |
```

Comment:
(for a followup PR)
we should likely rename / rethink this flag. I think we'll want something like:
`--projectTemplate` which takes a value of one of: `uwp`,  `winappsk-single`, `winappsdk-wap`, etc.

Reply:
filed https://github.com/microsoft/react-native-windows/issues/9711